src/os/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ set(libos_srcs
   filestore/WBThrottle.cc
   filestore/ZFSFileStoreBackend.cc
   memstore/MemStore.cc
-  kstore/kv.cc
+  bluestore/kv.cc
```

Comment:
@liewegas Shouldn't we have single kv.cc one folder up instead of duplicate versions at both kstore & bluestore?